headless/SDLHeadlessHost.cpp

```diff
@@ -166,6 +165,7 @@ bool GLDummyGraphicsContext::InitFromRenderThread(std::string *errorMessage) {
 	renderManager_->SetSwapFunction([&]() {
 		SDL_GL_SwapWindow(screen_);
 	});
+	// TODO: Support SwapInterval
```

Comment:
I'm not sure it's really necessary for headless either way, as long as it doesn't slow down and can still render.  Ideally it shouldn't be presenting exactly, I guess, or it should but should always be immediate since it'll be in fast forward mode?

Seems like headless crashed or something with these changes, though.  And that's even without an actual GPU, since it only runs softgpu on the GitHub runners.

-[Unknown]

Reply:
I think when adding this comment, I thought I was in a regular SDL backend, hah.